db/migrate/20221024071410_add_reporting_facility_state_region_indexes.rb

```diff
@@ -0,0 +1,8 @@
+class AddReportingFacilityStateRegionIndexes < ActiveRecord::Migration[6.1]
+  def change
+    add_index :reporting_facility_states, [:organization_region_id, :month_date], name: :index_fs_organization_month_date
+    add_index :reporting_facility_states, [:state_region_id, :month_date], name: :index_fs_state_month_date
+    add_index :reporting_facility_states, [:district_region_id, :month_date], name: :index_fs_district_month_date
+    add_index :reporting_facility_states, [:block_region_id, :month_date], name: :index_fs_block_month_date
```

Comment:
These lookups happen while refreshing org/state/district/block reports. Adding these on a BD prod-sized dataset reduced time by half, however the matview refresh time is expected to go up slightly.

Reply:
Do we need combination indices, or can we just add indices on org, state, district, and block id? I'm assuming month_date is already indexed? (If not, 😱 )

Reply:
The combination indexes make queries that query by both columns faster than two separate indexes. We usually always have a `month_date` filter in our queries, hence the combination indexes. 

We have a composite index for `month_date` and `facility_region_id`, so if someone makes a query with `month_date` alone, that one can be used.

Reply:
Nice, makes sense to me. Composite! That's the word I was looking for 😎